dumper-integration-tests/redshift-tests/src/main/java/com/google/edwmigration/dumper/base/TestBase.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.base;
+
+import static java.lang.String.format;
+import static java.lang.System.lineSeparator;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.LinkedHashMultiset;
+import com.opencsv.CSVParser;
+import com.opencsv.CSVParserBuilder;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Base class with general values for all TestNG test suites */
+public abstract class TestBase {
+
+  public static final CSVParser CSV_PARSER = new CSVParserBuilder().withEscapeChar('\0').build();
+  private static final Logger LOGGER = LoggerFactory.getLogger(TestBase.class);
+
+  /**
+   * @param dbMultiset List of extracted from DB items
+   * @param csvMultiset List of uploaded from Avro items
+   */
+  public static void assertMultisetsEqual(
+      LinkedHashMultiset dbMultiset, LinkedHashMultiset csvMultiset) {
+    LinkedHashMultiset dbMultisetCopy = LinkedHashMultiset.create(dbMultiset);
```

Comment:
This is a generic type, and should be parameterized.

Reply:
Done.

I made it in accordance with - https://docs.oracle.com/javase/tutorial/extra/generics/methods.html
>...type argument is being used for polymorphism; its only effect is to allow a variety of actual argument types to be used at different invocation sites. If that is the case, one should use wildcards.